hwdb/60-keyboard.hwdb

```diff
@@ -577,7 +577,8 @@ evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHP*ProBook*4*:pvr*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHewlett-Packard*:pnHPZBook*:pvr*
 evdev:atkbd:dmi:bvn*:bvr*:bd*:svnHP*:pnHPZBook*:pvr*
  KEYBOARD_KEY_81=f20                                    # Fn+F8; Microphone mute button, should be micmute
-
+ KEYBOARD_KEY_f8=wlan                                   # Wireless HW switch button
+ 
```

Comment:
Unnecessary white spaces in this blank line. This seems to cause the CI failures.

Reply:
You are right.
How should I proceed? revert, commit, and pull request again? Or something more clever?
It's the first time to do a such thing... Sorry

Reply:
Force-push is better. But in this case, adding a commit to remove the whitespace is also ok.